### PR TITLE
[KYUUBI #2085][FOLLOWUP] Fix the wrong path of `module:hive`

### PR DESCRIPTION
**Pull request by cfmcgrady**
### _Why are the changes needed?_

Fix the wrong path of `module:hive`

### _How was this patch tested?_

- https://github.com/cfmcgrady/incubator-kyuubi/pull/12

### CODE REVIEW

**Comment by cfmcgrady** [comment]
@yaooqinn @pan3793

File: .github/labeler.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@
   - "kyuubi-zookeeper/**/*"
 
 "module:hive":
-  - "kyuubi-hive-sql-engine/**/*"
+  - "externals/kyuubi-hive-sql-engine/**/*"
```
Comment (yaooqinn) [inline .github/labeler.yml:77]: we can also add beeline, jdbc etc.

**Review by ulysses-you — APPROVED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (cfmcgrady) [reply .github/labeler.yml:78]: done.

**Review by yaooqinn — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**